Sources/SwifterAuth.swift

```diff
@@ -158,24 +156,21 @@ public extension Swifter {
         })
     }
     
-    class func application(_ app: UIApplication, open url: URL, options: [UIApplication.OpenURLOptionsKey : Any]) -> Bool {
-        guard let sourceApplication = options[.sourceApplication] as? String else { return false }
-        let isSSO = sourceApplication.hasPrefix("com.twitter") || sourceApplication.hasPrefix("com.atebits")
-        let isWeb = sourceApplication.hasPrefix("com.apple") || sourceApplication == Bundle.main.bundleIdentifier!
-        if isSSO {
-            let notification = Notification(name: .swifterSSOCallback, object: nil, userInfo: [CallbackNotification.optionsURLKey: url])
-            NotificationCenter.default.post(notification)
-        } else if isWeb {
-            let notification = Notification(name: .swifterCallback, object: nil, userInfo: [CallbackNotification.optionsURLKey: url])
-            NotificationCenter.default.post(notification)
+    #endif
+    
+    @discardableResult
+    class func handleOpenUrlSSO(_ url: URL, callbackUrl: URL) -> Bool {
```

Comment:
I would prefer if this is merged with `handleOpenUrl(_:callbackUrl:)`. Something like the following so that it defaults the the existing implementation without duplicating it.

```swift
class func handleOpenUrl(_ url: URL, callbackUrl: URL, isSSO: Bool = false) {
  // ...
}
```

Reply:
Fixed!
https://github.com/mattdonnelly/Swifter/pull/271/commits/398c0eab50c544102e42ec9ea75a2d8dbc9c00bb

Add isSSO parameter for handleOpenURL.